fiftyone/core/session.py

```diff
@@ -35,7 +43,7 @@
 _subscribed_sessions = defaultdict(set)
 
 
-def launch_app(dataset=None, view=None, port=5151, remote=False):
+def launch_app(dataset=None, view=None, port=5151, remote=False, window=None):
```

Comment:
Suggestion: always print a message when `launch_app()` runs that explains what to expect:

- `desktop=False` (browser): open localhost:<port> in your browser
- `desktop=True` (desktop): FiftyOne desktop App has launched
- notebook: run `session.show()` to open the App in a cell